Update modernjs

ScriptedAlchemy opened the pull request:
Updated versions of multiple dependencies including 'tslib', '@modern-js/runtime', '@modern-js/app-tools', and '@modern-js/eslint-config' in pnpm-lock.yaml. This will ensure better compatibility and performance of the project. Some peer dependencies like 'webpack' and '@types/react' are also brought up-to-date to enhance stability.

ScriptedAlchemy committed:
Update modernjs

Updated versions of multiple dependencies including 'tslib', '@modern-js/runtime', '@modern-js/app-tools', and '@modern-js/eslint-config' in pnpm-lock.yaml. This will ensure better compatibility and performance of the project. Some peer dependencies like 'webpack' and '@types/react' are also brought up-to-date to enhance stability.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/module-federation/module-federation-examples?pullRequest=3543) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/module-federation/module-federation-examples?pullRequest=3543) before we can accept your contribution.<br/><hr/>**ScriptedAlchemy** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/module-federation/module-federation-examples?pullRequest=3543) it.</sub>

ScriptedAlchemy committed:
Add @modern-js/builder-rspack-provider to package.json

Added '@modern-js/builder-rspack-provider' version 2.45.0 to deps in package.json files across various apps. This enhances build capabilities and improves project configuration by ensuring it uses the latest library version.
